docs/Remove-Demo-Code.md

```diff
@@ -0,0 +1,68 @@
+# Remove Demo Code
+
+Whenever users use command line boilerplate tool like `ignite` or `create-react-app`, the first thing that most users do is delete all the demo code to start their project.
+
+Instead of going through the files yourself, or using find and replace, run `npx ignite-cli remove-demo`.
+
+After the command, the Ignite boilerplate will now have the smallest amount of demo code possible, while having all of Ignite's set up still at your fingertips.
+
+## Contributors
+
+When adding demo code to the boilerplate, use the following comments to have fine grained control over what gets stripped out. The goal is to be able to remove as much source code as possible while still having the generated Ignite app build correctly.
+
+### `// @demo remove-file`
+
+Remove the entire file.
+
+#### Example:
+
+```tsx
+// @demo remove-file
+export * from "./Text"
+export * from "./Screen"
+```
+
+### `// @demo remove-current-line`
+
+Remove the current line from the source code.
+
+#### Example:
+
+```tsx
+const $style: ViewStyle = { padding: 10 } // @demo remove
```

Comment:
```suggestion
const $style: ViewStyle = { padding: 10 } // @demo remove-current-line
```

Reply:
Does it make sense to make this `@demo remove-line` instead? Shorter but still understandable?

Reply:
The main reason I decided to go with `@demo remove-current-line` is because if we use `@demo remove`, then our logic for checking other comments like `@demo remove-next-line` vs `@demo remove` gets more complicated that `file.includes(comment)`. Essentially, each `@demo remove` keyword is a unique id for an operation

Reply:
Ah, I misread the diff. I'll update this to remove to `@demo remove-current-line`. It's a little more verbose, but I feel like it's a clearer distinction with `@demo remove-next-line`.